Update index.md (#114)
amplitude/amplitude-dev-center: docs/data/sdks/android-kotlin/index.md

    Amplitude amplitude = new Amplitude(configuration);
    ```

> By default, Amplitude sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. The sessions are tracked by using `session_id`.

Reviewer: ```suggestion
By default, Amplitude sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. Amplitude tracks these sessions with`session_id`.
```

> By default, Amplitude sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. Even though these events aren't sent, sessions are still tracked by using `session_id`.

Reviewer: @zheniasemenina This paragraph is unclear. First sentence says that Amplitude sends those events, but the second says otherwise. Can you confirm which is correct?

Author: @markzegarelli This is incorrect, but it stems from the previous version of this SDK: it used to say '_By default, '[Amplitude] Start Session' and '[Amplitude] End Session' events **aren't sent**. Even though these events.._' 
So we can get rid of this piece now.

> By default, Amplitude sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. Even though these events aren't sent, sessions are still tracked by using `session_id`.

Author: ```suggestion
By default, Amplitude sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. The sessions are tracked by using `session_id`.
```
